.github/actions/setup/action.yml

```diff
@@ -11,7 +11,6 @@ runs:
     - uses: pnpm/action-setup@v4
       name: Install pnpm
       with:
-        version: 8
```

Comment:
The action also looks at the `packageManager` field. One less location to update the desired pnpm version.